Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use github pages for docker image
-FROM ghcr.io/jorgensd/cbcbeatx:v0.1.0
+FROM ghcr.io/jorgensd/cbcbeatx:v0.7.0
```

Comment:
So, we will follow the version of dolfinx?

Reply:
For now, yes. Might change once we start adding more features in another pace than dolfinx.